wpilibc/src/test/native/cpp/geometry/Twist2dTest.cpp

```diff
@@ -15,29 +15,30 @@ using namespace frc;
 static constexpr double kEpsilon = 1E-9;
 
 TEST(Twist2dTest, Straight) {
-  const Twist2d straight{5.0, 0.0, 0.0};
+  const Twist2d straight{5.0_m, 0.0_m, 0.0_rad};
   const auto straightPose = Pose2d().Exp(straight);
 
-  EXPECT_NEAR(straightPose.Translation().X(), 5.0, kEpsilon);
-  EXPECT_NEAR(straightPose.Translation().Y(), 0.0, kEpsilon);
-  EXPECT_NEAR(straightPose.Rotation().Radians(), 0.0, kEpsilon);
+  EXPECT_NEAR(straightPose.Translation().X().to<double>(), 5.0, kEpsilon);
+  EXPECT_NEAR(straightPose.Translation().Y().to<double>(), 0.0, kEpsilon);
+  EXPECT_NEAR(straightPose.Rotation().Radians().to<double>(), 0.0, kEpsilon);
 }
 
 TEST(Twist2dTest, QuarterCircle) {
-  const Twist2d quarterCircle{5.0 / 2.0 * 3.14159265358979323846, 0,
-                              3.14159265358979323846 / 2.0};
+  const Twist2d quarterCircle{5.0_m / 2.0 * 3.14159265358979323846, 0_m,
+                              3.14159265358979323846_rad / 2.0};
```

Comment:
Use wpi::math::pi for this. I'm assuming `(wpi::math::pi)_rad` would work.

Reply:
Using the literal didn't work here so I wrapped it with a `units::radian_t`